tests/providers/amazon/aws/transfers/test_s3_to_sql.py

```diff
@@ -0,0 +1,89 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest.mock import patch
+
+from sqlalchemy import or_
+
+from airflow import configuration, models
+from airflow.providers.amazon.aws.transfers.s3_to_sql import S3ToSqlOperator
+from airflow.utils import db
+from airflow.utils.session import create_session
+
+
+class TestS3ToSqlTransfer(unittest.TestCase):
```

Comment:
For net-new tests, it's preferred to use `pytest` rather than `unittest`.

Reply:
@thinhnd2104 A friendly reminder to have this new test use `pytest` instead of `unittest`. No rush of course.